src/parser/paladin/holy/modules/azeritetraits/GlimmerOfLight.js

```diff
@@ -108,42 +134,75 @@ class GlimmerOfLight extends Analyzer {
           </>
         )}
         tooltip={(
-          <>
+          <Trans>
             Total healing done: <b>{formatNumber(this.totalHealing)}</b><br />
             Beacon healing transfered: <b>{formatNumber(this.healingTransfered)}</b><br />
+            Glimmer damage: <b>{formatNumber(this.damage)}</b><br />
             Holy Shocks/minute: <b>{this.holyShocksPerMinute.toFixed(1)}</b><br />
             Early refresh(s): <b>{this.earlyRefresh}</b><br />
-            Lost to early refresh: <b>{(this.wasted/1000).toFixed(1)}(sec) {(this.glimmersWasted * 100).toFixed(1)}%</b><br />
-            Glimmer damage: <b>{formatNumber(this.damage)}</b><br />
-          </>
+            Lost to early refresh: <b>{(this.wastedEarlyRefresh/1000).toFixed(1)}(sec) {(this.earlyGlimmersWasted * 100).toFixed(1)}%</b><br />
+            Glimmer of Lights over {GLIMMER_CAP} buff cap: <b>{this.overCap}</b><br />
+            Lost to over capping: <b>{(this.wastedOverCap/1000).toFixed(1)}(sec) {(this.overCapGlimmersWasted * 100).toFixed(1)}%</b><br />
+          </Trans>
         )}
       />
     );
   }
 
-  get suggestedGlimmerUsage() {
+
+  get suggestEarlyRefresh() {
     return {
-      actual: this.glimmersWasted,
+      actual: this.earlyGlimmersWasted,
       isGreaterThan: {
         minor: 0.15,
         average: 0.25,
-        major: .35,
+        major: 0.35,
+      },
+      style: 'percentage',
+    };
+  }
+
+  get suggestGlimmerCap() {
+    return{
+      actual: this.overCapGlimmersWasted,
+      isGreaterThan: {
+        minor: 0.1,
+        average: 0.2,
+        major: 0.3,
       },
       style: 'percentage',
     };
   }
 
   suggestions(when) {
-    when(this.suggestedGlimmerUsage).addSuggestion((suggest, actual, recommended) => {
-      return suggest(
-        <>
-          Your usage of <SpellLink id={SPELLS.GLIMMER_OF_LIGHT.id} /> can be improved. Try to avoid overwritting buffs too early.
-        </>,
-      )
+    if (this.owner.builds.GLIMMER.active){
+      when(this.suggestEarlyRefresh).addSuggestion((suggest, actual, recommended) => {
+        return suggest(
+          <Trans>
+            Your usage of <SpellLink id={SPELLS.GLIMMER_OF_LIGHT.id} /> can be improved. Try to avoid overwritting buffs too early.
+          </Trans>,
+        )
+          .icon(SPELLS.GLIMMER_OF_LIGHT.icon)
+          .actual(`Percentage uptime lost to early refresh was ${formatPercentage(this.earlyGlimmersWasted)}%`)
```

Comment:
I think this is worded slightly vague. Casting HS while already capped will just apply a new buff, right? So no actual uptime of what was max feasible with the cap will be lost. How about:

```suggestion
        .actual(`${formatPercentage(this.overCapGlimmersWasted)}% buff duration dropped due to buff cap`)
```

Reply:
> Holy Shock is still the strongest heal even without Glimmer, is it not?  @MartijnHols 

Here is a top [mythic log](https://www.warcraftlogs.com/reports/6MbwHrG1Whkgj3Kp#fight=24&type=healing&source=6) from a kill in August, Glimmer is doing 32.70% while Holy Shock is doing 22.22%.  Even in [lower level play](https://www.warcraftlogs.com/reports/4qWtBdMw2TvNhpJX#fight=29&type=healing&source=15) when glimmer is doing 38.6% overhealing it is still doing as much healing to Holy Shock.  The goal is always to cast as many Holy Shocks as possible, but you also need to be mindful of who already has a glimmer and who does not.  Correct play will result in a number of early refreshes, to get the most benefit from Glimmer of Light you want to stay as close to 8 active glimmers for as much of the fight as possible.

> Casting HS while already capped will just apply a new buff, right? So no actual up-time of what was max feasible with the cap will be lost.

I agree that the wording needs some clarity, however I don't think combining the suggestions is the solution.  The suggestion starting at line 191 counts the glimmers cast over the cap, the first suggestion is for counting early refreshes.  These are two separate items that can be improved upon.   With current haste levels you are only getting above the 8 Glimmer cap by stacking all your haste cooldowns.  Moving into 8.3 you will want to stagger your cooldowns to stay at cap for the majority of the fight, the current play style is to get 15+ glimmers rolling into major damage events then holding CDs for the next big raid damage.  The suggestion is there to remind players of the cap and ensure they change their play accordingly.

It is true that when you already have 8 glimmers up an early refresh actually the same as recasting on a previous target.  I will swap the order of the if statements accordingly.

Reply:
I have changed the statistic to count all Holy Shocks cast with 8 glimmers up as an over-cap cast.  I have therefore increased the cap suggestion threshold to 15% as it will be counting more glimmers than the previous method.

I made some changes to the suggestion wording in the suggestion.
![image](https://user-images.githubusercontent.com/3276248/71705963-19e2a800-2db0-11ea-983c-bd0180ae6ca4.png)

Both the suggestion text and the suggestion threshold prioritize healing low health targets, but if >%30 of your Holy Shocks are going to the same person it is leaving a lot of potential healing on the table and should be pointed out.  (Everyone has that one guildie who is always standing in the fire, but 30% is a little too much).